hack/ci/Makefile

```diff
@@ -3,7 +3,7 @@ PROJECT_COMMON ?= ${PROJECT_ROOT}/hack/common
 
 MODULECHARTTEST ?= ${PROJECT_ROOT}/module-chart-test
 
-include ${PROJECT_ROOT}/.env
+include ${PROJECT_ROOT}/.version
```

Comment:
Before the change and renming `.$(FILE)` it has following content:
```
ifndef MODULE_VERSION
    include .version
endif
```
I think that we should replace `${PROJECT_ROOT}/.version`  with the content of the file